include/sdf/Capsule.hh

```diff
@@ -72,6 +75,9 @@ namespace sdf
     /// \return A reference to a gz::math::Capsuled object.
     public: gz::math::Capsuled &Shape();
 
+    /// \brief Calculate an return the Mass Matrix values for the Capsule
+    public: std::optional< gz::math::MassMatrix3d >  MassMatrix(const double density);
```

Comment:
Same comment as above

Reply:
Corrected in b092904db4ef5c491672619076e75f4509b3369b